frontend/components/LastUpdatedHostCount/LastUpdatedHostCount.tests.tsx

```diff
@@ -17,19 +18,22 @@ describe("Last updated host count", () => {
     expect(hostCount).toBeInTheDocument();
     expect(updateText).toBeInTheDocument();
   });
-  it("renders never if missing timestamp", () => {
-    render(<LastUpdatedHostCount />);
-
-    const text = screen.getByText("Updated never");
 
-    expect(text).toBeInTheDocument();
+  it("renders 'Updated never' if lastUpdatedAt is explicitly null", () => {
+    render(<LastUpdatedHostCount lastUpdatedAt={null} />);
+    expect(screen.getByText("Updated never")).toBeInTheDocument();
   });
 
-  it("renders tooltip on hover", async () => {
-    render(<LastUpdatedHostCount hostCount={0} />);
-
-    await fireEvent.mouseEnter(screen.getByText("Updated never"));
+  it("does not render updated text if lastUpdatedAt is undefined", () => {
+    render(<LastUpdatedHostCount />);
+    expect(screen.queryByText(/Updated/i)).not.toBeInTheDocument();
+  });
 
+  it("renders tooltip on hover when 'Updated never'", async () => {
+    const { user } = renderWithSetup(
```

Comment:
(Side fix) Newer established frontend pattern is to use user from renderWithSetup to better mock interacting with UI